docs/source/changelog.md

```diff
@@ -14,6 +14,14 @@ changes in pull requests], this list should be updated.
 
 ## 4.0
 
+### 4.0.0-beta.4 - 2024-10-11
+
+#### Maintenance and upkeep improvements
+
```

Comment:
```suggestion

-  Update ldapauthenticator from 1.3.2 to 2.0.0.b2 [#126](https://github.com/manics/zero-to-jupyterhub-k8s/pull/126)
```
If we get the ldapauthenticator beta in this means all major components changes will be in a Z2JH beta.

Reply:
Oooh, this was meant to be part of the beta.1 already, i messed up.

Reply:
Updated and pushed a commit to include this with a notice about this mistakenly being announced to be done in beta.1 but wasn't actually part of a release until now.